robottelo/ssh.py

```diff
@@ -241,6 +241,22 @@ def upload_file(local_file, remote_file, key_filename=None, hostname=None):
             sftp.close()
 
 
+def upload_files(local_dir, remote_dir, file_type='.txt'):
+    """ Upload all files from directory to a remote directory
+    :param local_dir: all files from local path to be uploaded.
+    :param remote_dir: a remote path where the uploaded files will be
+           placed.
+    :param file_type: filter only files contains the type extension
+    """
+    command("mkdir -p {}".format(remote_dir))
+    for root, dirs, files in os.walk(local_dir):
+        for local_filename in files:
+            if file_type in local_filename:
+                remote_filename = "{0}{1}".format(remote_dir, local_filename)
```

Comment:
Do you want to maintain the local directory structure? Because now you're losing it, potentially overwriting files. Let's say you have `{local_dir}/subdir1/file1.txt` and `{local_dir}/subdir2/file1.txt` this ends up as `{remote_dir}file1.txt` with this code. You probably want `os.path.join(remote_dir, os.path.relpath(root, local_dir), local_filename)` if you want to maintain the directory structure.

Reply:
Yes, will do this change